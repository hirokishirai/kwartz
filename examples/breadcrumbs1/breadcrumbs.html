<html>
  <body>

    <div id="mark:breadcrumbs">
      <a href="/" id="mark:item">top</a>
      <span id="mark:separator"> &lt; </span>
      <a href="/kwartz">kwartz</a> &lt;
      <a href="/kwartz/examples">examples</a> &lt;
      breadcrumbs
    </div>

  </body>
</html>
