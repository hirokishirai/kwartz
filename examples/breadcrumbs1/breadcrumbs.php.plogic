#breadcrumbs {
  logic: {
    $i = 0;
    foreach ($breadcrumbs as $item) {
      if (++$i == 1) {
        _element('separator');
      }
      _element('item');
    }
  }
}  

#item {
  value:  $item['title'];
  attrs:  "href" $item['path'];
  logic: {
      if ($item['path']) {
        _stag();
        _cont();
        _etag();
      } else {
        _cont();
      }
  }
}

