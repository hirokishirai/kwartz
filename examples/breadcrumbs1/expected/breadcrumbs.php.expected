<html>
  <body>

      <a href="/">TOP</a> &lt;       <a href="/kwartz">Kwartz</a> &lt;       <a href="/kwartz/examples">Examples</a> &lt; breadcrumbs
  </body>
</html>
